app/helpers/sort_helper.rb

```diff
@@ -270,4 +278,14 @@ def sort_header_tag(column, options = {})
 
     content_tag('th', sort_link(column, caption, default_order, lang: lang), options)
   end
+
+  # Returns a table header tag similar to +sort_header_tag+, but
+  # according to the LSG table component.
+  def sort_header_tag_with_lsg(column, options = {})
```

Comment:
Just me nagging about naming things:

`with_lsg` sounds like you would include a living style guide... I see that you won't be able to replace the old method.
Probably you could call it `styled_sort_header_tag`? I saw those `styled_*` helpers a lot lately, but I only assume that they are related to being styled in the new fancy way...